configmaps.tf

```diff
@@ -24,3 +53,200 @@ resource "kubernetes_config_map" "misarch_catalog_env_vars" {
+  }
+}
+
+resource "kubernetes_config_map" "misarch_order_env_vars" {
+  metadata {
+    name      = local.misarch_order_env_vars_configmap
+    namespace = local.namespace
+  }
+
+  data = {
+    "SPRING_R2DBC_URL"      = "r2dbc:postgresql://${local.order_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_FLYWAY_URL"     = "jdbc:postgresql://${local.order_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_R2DBC_USERNAME" = var.MISARCH_DB_USER
+    "SPRING_R2DBC_PASSWORD" = random_password.misarch_order_db_password.result
+  }
+}
+
+resource "kubernetes_config_map" "misarch_payment_env_vars" {
+  metadata {
+    name      = local.misarch_payment_env_vars_configmap
+    namespace = local.namespace
+  }
+
+  data = {
+    "SPRING_R2DBC_URL"      = "r2dbc:postgresql://${local.payment_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_FLYWAY_URL"     = "jdbc:postgresql://${local.payment_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_R2DBC_USERNAME" = var.MISARCH_DB_USER
+    "SPRING_R2DBC_PASSWORD" = random_password.misarch_payment_db_password.result
+  }
+}
+
+resource "kubernetes_config_map" "misarch_review_env_vars" {
+  metadata {
+    name      = local.misarch_review_env_vars_configmap
+    namespace = local.namespace
+  }
+
+  data = {
+    "SPRING_R2DBC_URL"      = "r2dbc:postgresql://${local.review_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_FLYWAY_URL"     = "jdbc:postgresql://${local.review_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_R2DBC_USERNAME" = var.MISARCH_DB_USER
+    "SPRING_R2DBC_PASSWORD" = random_password.misarch_review_db_password.result
+  }
+}
+
+resource "kubernetes_config_map" "misarch_return_env_vars" {
+  metadata {
+    name      = local.misarch_return_env_vars_configmap
+    namespace = local.namespace
+  }
+
+  data = {
+    "SPRING_R2DBC_URL"      = "r2dbc:postgresql://${local.return_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_FLYWAY_URL"     = "jdbc:postgresql://${local.return_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_R2DBC_USERNAME" = var.MISARCH_DB_USER
+    "SPRING_R2DBC_PASSWORD" = random_password.misarch_return_db_password.result
+  }
+}
+
+resource "kubernetes_config_map" "misarch_shipment_env_vars" {
+  metadata {
+    name      = local.misarch_shipment_env_vars_configmap
+    namespace = local.namespace
+  }
+
+  data = {
+    "SPRING_R2DBC_URL"      = "r2dbc:postgresql://${local.shipment_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_FLYWAY_URL"     = "jdbc:postgresql://${local.shipment_db_url}/${var.MISARCH_DB_DATABASE}"
+    "SPRING_R2DBC_USERNAME" = var.MISARCH_DB_USER
+    "SPRING_R2DBC_PASSWORD" = random_password.misarch_shipment_db_password.result
+  }
+}
+
+resource "kubernetes_config_map" "misarch_shoppingcart_env_vars" {
+  metadata {
+    name      = local.misarch_shoppingcart_env_vars_configmap
+    namespace = local.namespace
+  }
+
+  data = {
+    "SPRING_R2DBC_URL"      = "r2dbc:postgresql://${local.shoppingcart_db_url}/${var.MISARCH_DB_DATABASE}"
```

Comment:
these variables make no sense for the rust services

Reply:
Done